framework/src/node/forger/forger.ts

```diff
@@ -223,30 +237,22 @@ export class Forger {
 		if (!isSyncedWithNetwork(lastBlockHeader, forgingInput)) {
 			throw new Error('Failed to enable forging as the node is not synced to the network.');
 		}
+		const forgerInfo = previouslyForgedMap.get(forgerAddress);
+		if (overwrite !== true) {
+			if (forgerInfo !== undefined && !IsEqualForgingInfo(forgerInfo, forgingInput)) {
+				throw new Error('Failed to enable forging due to contradicting forger info.');
```

Comment:
should we log or respond back with `forgerInfo, forgingInput`? just for debugging or information purpose